src/coprocessor/dag/expr/scalar_function.rs

```diff
@@ -298,6 +298,7 @@ impl ScalarFunc {
             | ScalarFuncSig::CaseWhenReal
             | ScalarFuncSig::CaseWhenString
             | ScalarFuncSig::Concat
+			| ScalarFuncSig::ConcatWS
```

Comment:
ditto

Reply:
Yes!  I will change those. Thank you  again.

Reply:
haha！